src/styles/modules/email-form.scss

```diff
@@ -0,0 +1,20 @@
+.email-form {
+  display: flex;
+  margin-bottom: $base-spacing-unit-small;
+  color: $grey-1;
+  input[type=email] {
+    -webkit-appearance: none;
+    border-radius: 0;
+    border-right: 0;
+    border-bottom-left-radius: 4px;
+    border-top-left-radius: 4px;
+    border: 1px solid $grey-3;
+    border-right: 0;
+    padding: $base-spacing-unit $base-spacing-unit-tiny;
+    width: 100%;
+  }
+  .button {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
```

Comment:
Can shorthand be used?

Reply:
This extends the `.button` class that uses explicit border-radius parameters. It's not using shorthand to allow preservation of the setting (if it ever changes)